config.js

```diff
@@ -12,7 +12,7 @@ var config = {
     'scss_dirs': [
       // Must have the top level path first.
       __dirname + '/jujugui/static/gui/src/app/assets/css/',
-      __dirname + '/jujugui/static/gui/src/app/assets/css/machine-view'
+      __dirname + '/jujugui/static/gui/src/app/assets/css/canvas'
```

Comment:
I'm not sure if we still use it. I know in the past if a dir wasn't here the css would not be watched for changes and at some point I added a canvas dir and forgot to add it here which I noticed when removing the machine-view dir.